File: 75-sort-colors/sort-colors.js
/**
 * @param {number[]} nums
 * @return {void} Do not return anything, modify nums in-place instead.
 */

 nums = [2,0,2,1,1,0]
var sortColors = function(nums) {
    
for (i = 0; i < nums.length; i++) {

    for (j = i + 1; j < nums.length; j++) {
        let temp = 0;
        if (nums[i] > nums[j]) {
            temp = nums[i]
            nums[i] = nums[j]
            nums[j] = temp
        }
    }

}
return nums


    }


    

File: 75-sort-colors/sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int n = nums.size();
        map<int, int> mpp;

        for (int i = 0; i < n; i++) {
            mpp[nums[i]]++;
        }
        // cout << mpp[0]<<endl;
        for (int i = 0; i < mpp[0]; i++) {
            nums[i] = 0;
        }

        for (int i = mpp[0]; i < mpp[0] + mpp[1]; i++) {
            nums[i] = 1;
        }

        for (int i = mpp[0] + mpp[1]; i < n; i++) {
            nums[i] = 2;
        }
    }
};